dev-resources/logback.xml

```diff
@@ -5,11 +5,11 @@
         </encoder>
     </appender>
 
-    <logger name="puppetlabs" level="error"/>
+    <logger name="puppetlabs" level="trace"/>
```

Comment:
one downside to having it set to trace in logback-test would be that basically all tests would need to be wrapped in a with-test-logging call, which will make debugging non-trivial test failures a lot harder.